lib/adagrams.rb

```diff
@@ -0,0 +1,145 @@
+# Wave 1
+def draw_letters
+  raw_tiles = [
+  {letter: "A", weight: 9, score: 1},
+  {letter: "B", weight: 2, score: 3},
+  {letter: "C", weight: 2, score: 3},
+  {letter: "D", weight: 4, score: 2},
+  {letter: "E", weight: 12, score: 1},
+  {letter: "F", weight: 2, score: 4},
+  {letter: "G", weight: 3, score: 2},
+  {letter: "H", weight: 2, score: 4},
+  {letter: "I", weight: 9, score: 1},
+  {letter: "J", weight: 1, score: 8},
+  {letter: "K", weight: 1, score: 5},
+  {letter: "L", weight: 4, score: 1},
+  {letter: "M", weight: 2, score: 3},
+  {letter: "N", weight: 6, score: 1},
+  {letter: "O", weight: 8, score: 1},
+  {letter: "P", weight: 2, score: 3},
+  {letter: "Q", weight: 1, score: 10},
+  {letter: "R", weight: 6, score: 1},
+  {letter: "S", weight: 4, score: 1},
+  {letter: "T", weight: 6, score: 1},
+  {letter: "U", weight: 4, score: 1},
+  {letter: "V", weight: 2, score: 4},
+  {letter: "W", weight: 2, score: 4},
+  {letter: "X", weight: 1, score: 8},
+  {letter: "Y", weight: 2, score: 4},
+  {letter: "Z", weight: 1, score: 10}]
+
+  tile_set = []
+
+  # extracts letter and score keys and puts them in array called raw_tiles
```

Comment:
Clever use of the data structure you created. Consider also how you could use a data structure organized like this:  {"A"=>9, "B"=>2,...}